cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -548,7 +550,8 @@ public void run() {
    */
   public enum TaskState {
     ACTIVE("Active"),
-    COMPLETED("Completed");
+    COMPLETED("Completed"),
+    COMPLETED_WITH_ERROR("CompletedWithError");
```

Comment:
Can we add a comment before `_allUserTaskIdToFutureMap` to indicate that `COMPLETED` is also keeps track of tasks that completed with error?